tools/clang/include/clang/SPIRV/SpirvContext.h

```diff
@@ -294,6 +420,40 @@ class SpirvContext {
   // Major/Minor hlsl profile version.
   uint32_t majorVersion;
   uint32_t minorVersion;
+
```

Comment:
For every debug type or instruction that you have allocated memory for in the SpirvContext, you need to make sure the memory is released in the SpirvContext destructor. (and of course you need to make sure you don't double delete in case you are keeping the same pointer in multiple datastructures).

Reply:
done